### ISSUE TITLE
Warning caused by : Transport Delay block of mechanical subsystem

### ISSUE THREAD

**Issue by joernmalzahn** [geez0x1/CompliantJointToolbox#52]
The Simulink files throw this warning on the console:
```
Warning: When delay time is set to zero, the transport delay block
'Ex_08_PD_Feedforward_ClosedLoopDOB/JointMechanicalSubsystem/input_delay' is automatically set to support direct
feedthrough. This may cause an algebraic loop. A Memory Block can be used in place of the Transport Delay to break
the loop 
``` 
We should implement the delay, such that Simulink does not feel obliged to throw this warning.

**Comment by geez0x1** [comment]
Does this only happen in the DOBs? Probably a consequence of discretisation methods of the filters then.

**Comment by geez0x1** [comment]
Fixed in 2dc8ada.

The delay value inside the actual delay block is computed so that it is >0 when delay is DISABLED,
to avoid MATLAB throwing warnings unnecessarily. Depending on whether delay is actually enabled the signal is routed around the block or through it.

**Comment by geez0x1** [comment]
Also fixed for electrical model blocks in 8dd9c29ae34234d442db7da6dc4aee19a674f69e. Also checked the `jointBus` variables that are used: The electrical dynamics correctly use the true (`NoNoise`) velocity of the rotor, and the current controller uses the motor velocity variable that it should have access to (which may be polluted by delays, noise, etc).

**Comment by geez0x1** [comment]
Note this may still be an issue in rare cases for the output delays. Haven't consistently seen this though.
